### DISCUSSION TITLE
BRouter (external) guide

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#137] [Show and tell]
- Install BRouter app from [Google Play](https://play.google.com/store/apps/details?id=btools.routingapp)
Installiere die BRouter App aus [Google Play](https://play.google.com/store/apps/details?id=btools.routingapp)

-  Open BRouter and select storage location for the data and Download Manager
Öffne BRouter und wähle den Speicherort für die Daten und den Download-Manager

- Zoom and select the region for offline routing and start downloading
Zoome und wähle die Region für die Offline-Routenberechnung und starte den Download

<img src="https://user-images.githubusercontent.com/3484020/209925095-486c7eda-c244-4427-9369-f4bf7a7902d5.jpg" width="50%"/><img src="https://user-images.githubusercontent.com/3484020/234211747-5b0f8548-c36a-449a-9361-617a94e181c8.png" width="50%"/>
<img src="https://user-images.githubusercontent.com/3484020/234211768-ac394b6a-9815-413d-b9ce-9b28e1aa6e5e.png"/>
<img src="https://user-images.githubusercontent.com/3484020/234211790-c4ac583e-c0c3-46ed-9d53-21544cddcef2.png"/>

**Comment by 0709wiwiwi** [comment]
Brouter with profiles available as desired ?
I myself don't venture to edit these profiles.

Too many choices available also make it unclear.
You can't see the forest for the trees.
Still, here a source:
https://github.com/zossebart/brouter-misc

At Enduro I see an entry with gates, who tests ?

What I myself am currently testing is reroute.brf
Used in the Brouter app with foot_short
Reroute is tolerant with access rules. 
Without protest, any osm path to be used ?
If even that fails, only the beeline than remains.

**Comment by devemux86** [comment]
![Screenshot_1685369191](https://github.com/devemux86/cruiser/assets/3484020/c6461507-8bfd-46c0-b5b8-52b40b6156be)

|Color|Description|
|-|-|
|Gray|Regions with updates|
|Blue|Regions with no updates|
|Green|Download new regions|
|Yellow|Download updates|
|Red|Delete regions|

|Farbe|Beschreibung|
|-|-|
|Grau|Regionen mit Aktualisierungen|
|Blau|Regionen ohne Aktualisierungen|
|Grün|Neue Regionen herunterladen|
|Gelb|Aktualisierungen herunterladen|
|Rot|Regionen löschen|
